settings.gradle

```diff
@@ -0,0 +1,10 @@
+plugins {
+    id 'com.diffplug.blowdryerSetup' version '1.6.0'
+}
+
+apply plugin: 'com.diffplug.blowdryerSetup'
+
+blowdryerSetup {
```

Comment:
Since this does things completely unrelated to the text, please document it

`// Pulls the spotless config from the example mod repository` should do

Reply:
@combusterf guess this have to be added to example mod to add to all files in the future ?

Reply:
Changes to settings.gradle and build.gradle have to be PRd into the ExampleMod1.7.10 repository, because we may periodically overwrite all mods' repos copies of these files with ones taken from ExampleMod1.7.10

Reply:
we ned to do it to the example mod later